site/content/docs/Localisation/_index.md

```diff
@@ -0,0 +1,108 @@
+---
```

Comment:
i would move doc changes to separate PR to be merged closer to the release.

Reply:
I will split up the pr into two separate ones.